cmake-format.py

```diff
@@ -0,0 +1,10 @@
+with section("format"):
+    line_width = 105
+    tab_size = 4
+    dangle_parens = True
+    separate_ctrl_name_with_space = True
+    max_subgroups_hwrap = 2
+    max_prefix_chars = 4
+
+with section("markup"):
+    enable_markup = False
```

Comment:
This file was generated using `cmake-format --dump-config python`. It is passed into the action explicitly.

The markup section is ignored because its modifications to the comments were super destructive.